{"code":"ECONNREFUSED","fatal":true,"level":"error","message":"Error connecting to MySQL:","service":"notepad-app","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1122:18)\n    at afterConnectMultiple (node:net:1689:7)","timestamp":"2025-05-18 17:42:21"}
{"code":"ER_BAD_DB_ERROR","errno":1049,"level":"error","message":"Error connecting to MySQL: Unknown database 'notepad_db'","service":"notepad-app","sqlMessage":"Unknown database 'notepad_db'","sqlState":"42000","stack":"Error: Unknown database 'notepad_db'\n    at Packet.asError (C:\\Users\\Asif\\Desktop\\edtho\\node_modules\\mysql2\\lib\\packets\\packet.js:740:17)\n    at ClientHandshake.execute (C:\\Users\\Asif\\Desktop\\edtho\\node_modules\\mysql2\\lib\\commands\\command.js:29:26)\n    at Connection.handlePacket (C:\\Users\\Asif\\Desktop\\edtho\\node_modules\\mysql2\\lib\\base\\connection.js:475:34)\n    at PacketParser.onPacket (C:\\Users\\Asif\\Desktop\\edtho\\node_modules\\mysql2\\lib\\base\\connection.js:93:12)\n    at PacketParser.executeStart (C:\\Users\\Asif\\Desktop\\edtho\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\Asif\\Desktop\\edtho\\node_modules\\mysql2\\lib\\base\\connection.js:100:25)\n    at Socket.emit (node:events:518:28)\n    at addChunk (node:internal/streams/readable:561:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)\n    at Readable.push (node:internal/streams/readable:392:5)","timestamp":"2025-05-18 17:49:40"}
